### PR TITLE
Aqa/cypress assertions

### PR DESCRIPTION
**Pull request by manatskaya**
Hi Luibov,
Please review my pull request when you have a chance.
Thanks
Svitlana

### CODE REVIEW

File: cypress/e2e/app_tests/error_validation.cy.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+            cy.get('.form-control-group .caption.status-danger').as('error');
+            cy.get('@error').eq(0).should('contain', ' Full name should contains from 4 to 50 characters ');
+
+            cy.get('#input-email').focus().realType('test.com');
+            cy.get("body").click();
+            cy.get('@error').eq(1).should('contain', ' Email should be the real one! ');
+
+            cy.get('#input-password').focus().realType('1');
+            cy.get("body").click();
+            cy.get('@error').eq(2).should('contain', ' Password should contain from 4 to 50 characters ');
```
Comment (landriiuk) [inline cypress/e2e/app_tests/error_validation.cy.js]:
```suggestion

function verifyHeadercontent(selector, index, errorMessage ){
cy.get(selector).focus().realType('N');
cy.get("body").click();
cy.get('.form-control-group .caption.status-danger').as('error');
cy.get('@error').eq(index).should('contain', errorMessage);

}

verifyHeaderContent('#input-name', 0,  'Full name should contains from 4 to 50 characters ');
verifyHeaderContent('#input-email', 1,  ' Email should be the real one! '))
verifyHeaderContent('#input-password'', 2, ' Password should contain from 4 to 50 characters ')
```

Comment (landriiuk) [reply cypress/e2e/app_tests/error_validation.cy.js]: Just to simplify your code.

File: cypress/e2e/app_tests/form_filling.cy.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+            cy.get('#input-password').focus().realType('123456');
+            cy.get('input#input-password[ng-reflect-model]').should('have.value', '123456');
+
+            cy.get('#input-re-password').focus().realType('123456');
+            cy.get('input#input-re-password[ng-reflect-model]').should('have.value', '123456');
+
+            cy.get('.label .custom-checkbox').realClick()
+                .should('have.class', 'custom-checkbox checked');
+
```
Comment (landriiuk) [inline cypress/e2e/app_tests/form_filling.cy.js]: you can try to rewrite it using function, as example in previous comment

**Review by landriiuk — APPROVED** [review]
LGTM
Check comment , it will make your code more readable
Then merge this pr

File: cypress/e2e/app_tests/form_filling.cy.js (hunk 2 above)
Comment (manatskaya) [reply cypress/e2e/app_tests/form_filling.cy.js]: updated, thank you

File: cypress/e2e/app_tests/error_validation.cy.js (hunk 1 above)
Comment (manatskaya) [reply cypress/e2e/app_tests/error_validation.cy.js]: updated. 
Thank you

**Pull request merged**